{'project_id': '1211', 'year': '1989', 'x': 804000, 'y': 2031420, 'plot_id': '2376', 'strata': '41'}
Pandas(Index=0, project_id='1211', plot_id='2376', x=804000, y=2031420)
called process_plot
Pandas(Index=0, project_id='1211', plot_id='2376', x=804000, y=2031420)
2376
Directory '/efs/timesync/1989/audit/' already exists.
Data has been pickled and saved to '/efs/timesync/1989/audit/1989_2376_1211.p'.
.............................................................................Failed qa: 1989-05-15T15:51:14.4720250Z
...................................Failed qa: 1989-06-24T15:56:11.0710190Z
Failed qa: 1989-07-02T15:50:23.1920560Z
........................................................Failed qa: 1989-10-06T15:48:18.8140130Z
.......Data has been pickled and saved to '/efs/timesync/1989/audit/1989_2376_1211_qa_rejects.p'.
Success tracking goes here
